### PR TITLE
Run lint etc in GitHub Actions

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v2"
+
+      - name: "Install Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.8"
```
Comment (portante) [inline .github/workflows/quality.yml:29]: Why 3.8 for quality checks but kitting is 3.7?

File: .github/workflows/quality.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+      - name: "Install dependencies"
+        run: |
+          set -xe
+          python -VV
+          python -m site
+          python -m pip install -r requirements/tox.pip
```
Comment (portante) [inline .github/workflows/quality.yml:36]: How come you don't use `tox` to do these steps?

File: .github/workflows/quality.yml (hunk 1 above)
Comment (nedbat) [reply .github/workflows/quality.yml:29]: ¯\\_(ツ)_/¯

File: .github/workflows/quality.yml (hunk 2 above)
Comment (nedbat) [reply .github/workflows/quality.yml:36]: These are needed to get tox installed. Then the rest is handled by tox.

**Pull request merged**